nss-cohort-36/bangazon-api-virtual_magicians
# Jl next ticket

# Description
[Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.]
Closes #[Enter issue #]
updated customer model to include an address (location) so will need to make migrations and run migrate with python manage.py

## Type of change
Please delete options that are not relevant.

# Testing Instructions
Make sure to commit local changes in your branch before switching to master.
1. `git checkout master`
2. `git fetch --all`
3. `git checkout jl-next-ticket`
4. [Enter specific testing instructions here to approve]
1:  run makemigrations and migrate to add address column to customer table
2:  using the react side, you can fully test the customer edit to add or change an address to your current user
3:  in terminal, run python manage.py test virtualmagicians -v 2 (will say 3 tests running, but only 2 written) - scroll up through the error to find 2 dots & 1 E - this will show that the 2 tests written pass (per Jisie this is fine)

# Checklist:
